# topic: /tcp_robot_state. Published by 'sensor_hub_server'. Used by 'baseboard_node' and 'wheel_control_node'

bool wheels_enabled
uint8 battery_percent    # 电量百分比 [0-100]
bool is_charge
